tests/Endpoints/ZoneCacheTest.php

```diff
@@ -75,4 +76,106 @@ public function testCachePurge()
         $this->assertTrue($result);
         $this->assertEquals('023e105f4ecef8ad9ca31a8372d0c353', $zones->getBody()->result->id);
     }
+
+    public function testCachePurgeIncludingEnvironments()
+    {
+        $envResp = $this->getPsr7JsonResponseForFixture('Endpoints/getEnvironments.json');
+        $cacheResp = $this->getPsr7JsonResponseForFixture('Endpoints/cachePurge.json');
+        $mock = $this->getMockBuilder(\Cloudflare\API\Adapter\Adapter::class)->getMock();
+
+        $mock->expects($this->once())
+            ->method('get')
+            ->willReturn($envResp)
+            ->with(
+                $this->equalTo('zones/c2547eb745079dac9320b638f5e225cf483cc5cfdda41/environments'),
```

Comment:
php < 7.2 doesn't like commas here.

```suggestion
                $this->equalTo('zones/c2547eb745079dac9320b638f5e225cf483cc5cfdda41/environments')
```